src/DynamoCore/Graph/Workspaces/PackageDependencyInfo.cs

```diff
@@ -144,7 +158,18 @@ internal class PackageDependencyInfo : INodeLibraryDependencyInfo
         /// <summary>
         /// Indicates whether this package is loaded in the current session
         /// </summary>
-        public bool IsLoaded { get; set; }
+        [Obsolete("This property is obsolete, use PackageDependencyState property instead", false)]
+        public bool IsLoaded
+        {
+            get {
+                return this.State.Equals(PackageDependencyState.Loaded);
```

Comment:
you should be able to do `==`  as an enum is an int, I would expect it is a value type.

Reply:
https://stackoverflow.com/questions/17492179/c-sharp-enum-equals-vs

Reply:
Yes, changed it.